Create activemqproducer.py

RaghuvirDav9 opened the pull request.

RaghuvirDav9 committed:
Create activemqproducer.py

RaghuvirDav9 committed:
Create activemqconsumer.py

RaghuvirDav9 committed:
Create list_ec2.py

uses boto3 library to connect and get information from aws.

NayangiriGoswamiGit commented on activemqproducer.py:
```diff
@@ -0,0 +1,10 @@
+import stomp
+import random
+
+q=random.Random()
+
+conn = stomp.Connection([('localhost', 61613)])
+conn.connect('admin', 'password', wait=True)
+message='hello from producer 1 number'+str(q.random())
+conn.send(body=message, destination='/queue/test')
+conn.disconnect()
```
This should be a function, so that I can import anywhere where I want to publish a messge.

NayangiriGoswamiGit commented on list_ec2.py:
```diff
@@ -0,0 +1,7 @@
+import boto3
```
use ```click``` library, which will accept instance state as an command line argument, if the argument is ```Running``` than just lisr running instances, otherwise all.

NayangiriGoswamiGit replied:
@RaghuvirDav9 I think this is pending, right?

NayangiriGoswamiGit commented on list_ec2.py:
```diff
@@ -0,0 +1,7 @@
+import boto3
+
```
I asked you to write unittest tests for this script, where are those?

NayangiriGoswamiGit replied:
@RaghuvirDav9 I think this is pending, right?

NayangiriGoswamiGit reviewed (changes requested):
Thansk @RaghuvirDav9 , I have requsted some changes,
1. Every class/function should have doc string in you code now onwards.
2. Every function parameter should have type info and return type info. Use ```typing``` library for that.

RaghuvirDav9 committed:
Update activemqproducer.py

updated the code

RaghuvirDav9 committed:
Update activemqconsumer.py

updated activemqconsumer

RaghuvirDav9 committed:
Update list_ec2.py

updated comment on this program

RaghuvirDav9 committed:
Create upload_zip_to_aws_s3.py

python script to zip and upload a directory to aws-s3

RaghuvirDav9 committed:
Update list_ec2.py

used click library to list running state instances ,passing 'running' as command argument

RaghuvirDav9 committed:
Create list_instances.py

this python script takes command line argument for the current state of EC2 instance and lists instances of that type

RaghuvirDav9 committed:
Create test_function.py

this python code is used for testing list_instances.py file for running and stopped tests

RaghuvirDav9 committed:
Create one_ec2.yaml

this yaml file when run on aws cli creates a single aws ec2 instance .
